ci/lib.sh

```diff
@@ -57,13 +57,20 @@ curl() {
 # This will contain the artifacts we want.
 # https://developer.github.com/v3/actions/workflow-runs/#list-workflow-runs
 get_artifacts_url() {
-  local head_sha
   local artifacts_url
-  head_sha=$(git rev-parse HEAD)
-  artifacts_url=$(curl -fsSL 'https://api.github.com/repos/cdr/code-server/actions/workflows/ci.yaml/runs?status=success&event=pull_request' | jq -r ".workflow_runs[] | select(.head_sha == \"$head_sha\") | .artifacts_url" | head -n 1)
+  local code_server_version
+  local version_branch
+  local workflow_runs_url
+  workflow_runs_url="https://api.github.com/repos/cdr/code-server/actions/workflows/ci.yaml/runs?status=success&event=pull_request"
+  code_server_version=$(node -pe "require('./package.json').version")
```

Comment:
Looks like we can use `$VERSION` (set below).

Reply:
LOL 🤦‍♂️ didn't think to scroll down in my editor 😂 Will change